### PR TITLE
Changes to README.md file for setting up a new install for the member

### PR DESCRIPTION
**Pull request by vivekkr**
Changes to README.md file for setting up a new install for the member

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,17 +29,19 @@ To get started working on Membership you will need to complete the following ste
...
 127.0.0.1   mem.thegulocal.com
 ```
 
-1. Run `./nginx/setup.sh`
+1. Run `membership-frontend$ ./nginx/setup.sh`
 1. Download our private keys from the `membership-private` S3 bucket. You will need an AWS account so ask another dev. If you have the AWS CLI set up you can run:
 
 ```
-aws s3 cp s3://membership-private/DEV/membership-keys.conf /etc/gu
+aws s3 cp s3://membership-private/DEV/membership-keys.conf /etc/gu --profile membership
```
Comment (tudorraul) [inline README.md]: This command is dependent on the configuration of ~/.aws/credentials, so setting the `profile` argument is optional. Please indicate this in the readme.

File: README.md [hunk 2]
```diff
@@ -101,14 +105,19 @@ Run through the set up instructions; once complete you will need to run:
 
 **theguardian.com frontend repo**: [https://github.com/guardian/frontend](https://github.com/guardian/frontend)
 
-Run through the set up instructions - note you need to make sure that your Frontend is set up to point at your _local_ Identity, not the `CODE` Identity, which means adding this to your `frontend.properties`:
+Run through the set up instructions - download `frontend` and make sure that your `frontend` project is set up to point at your _local_ Identity, not the `CODE` Identity, which means adding this to your `frontend.properties`:
+
+ `$ cd ~/.gu`
```
Comment (tudorraul) [inline README.md]: `cd` and `vi` can be replaced with a single command

Comment (tudorraul) [inline README.md]: `$` sign is not a part of the command and may differ on different platforms, I would not use it

**Comment by tudorraul** [comment]
:+1: bar the comments

File: README.md [hunk 3]
```diff
@@ -29,17 +29,19 @@ To get started working on Membership you will need to complete the following ste
...
+1. Change the ownership of the 'gu' directory under 'etc' to current user.
+   `$ sudo -i chown -R {username} /etc/gu`
 1. Add the following to your hosts file in `/etc/hosts`:
 
 ```
 127.0.0.1   mem.thegulocal.com
 ```
 
-1. Run `./nginx/setup.sh`
+1. Run `membership-frontend$ ./nginx/setup.sh`
```
Comment (davidrapson) [inline README.md]: Not sure what this is referring too?

File: README.md [hunk 4]
```diff
@@ -48,8 +50,10 @@ In `~/.aws/credentials` add the following:
 
 ```
 [membership]
-aws_access_key_id=[YOUR_AWS_ACCESS_KEY]
-aws_secret_access_key=[YOUR_AWS_SECRET_ACCESS_KEY]
+aws_access_key_id = [YOUR_AWS_ACCESS_KEY]
+aws_secret_access_key = [YOUR_AWS_SECRET_ACCESS_KEY]
+region = eu-west-1
```
Comment (davidrapson) [inline README.md]: Did you end up needing this? This should be inherited from the default settings below in `~/.aws/config`. Interesting if you did, though.

**Comment by davidrapson** [comment]
:+1: from me too, aside from a couple of queries. Nice to keep this updated.

File: README.md (hunk 2 above)
Comment (vivekkr) [inline README.md]: Incorporated `cd` and `vi`
Removed `$` sign

File: README.md (hunk 3 above)
Comment (vivekkr) [inline README.md]: Split into two to make it abundantly clear

File: README.md (hunk 4 above)
Comment (vivekkr) [inline README.md]: Needed the credentials for it to run. Added Region though so that its clear

**Comment by davidrapson** [comment]
:100: with the tweaks.

**Pull request merged**